pkg/handler/aws/participation/participation.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"github.com/CollActionteam/collaction_backend/auth"
+	"github.com/CollActionteam/collaction_backend/internal/constants"
+	m "github.com/CollActionteam/collaction_backend/internal/models"
+	"github.com/CollActionteam/collaction_backend/internal/participation"
+	"github.com/CollActionteam/collaction_backend/models"
+	"github.com/CollActionteam/collaction_backend/pkg/repository"
+	"github.com/CollActionteam/collaction_backend/pkg/repository/aws"
+	"github.com/CollActionteam/collaction_backend/utils"
+	"github.com/aws/aws-lambda-go/events"
+	"net/http"
```

Comment:
these imports are not formated

Reply:
Can you send me the correct way?
When I put in a different order my IDE shows me this.
<img width="740" alt="Screen Shot 2022-02-13 at 12 21 05" src="https://user-images.githubusercontent.com/18128330/153760067-a091a154-65fa-437c-8715-bfef11fa4c36.png">
If there is a specific configuration for the imports please share with me.